website/docs/api/candidate.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Candidate
```

Comment:
I'd be slightly more in favour of keeping `Candidate` with `KnowledgeBase`. Then if at some point we want a hierarchy of candidates, specific to certain knowledge bases, we can also document the specific KB + the specific Candidate together on their own page.

Reply:
Added back in.